### PR TITLE
319 item view

### PR DESCRIPTION
**Pull request by cdmo**
WIP, meaning don't merge. But, I'd like a first review. What's missing is tests. Also missing is transforming the Bookmark action into a member of the record-level action link button bar.

Additional change: I moved the pivot facet sass out of psulib_icons.

Semi-related: I started using better_errors, it is _better_ than web_console. Thinking we should replace it in the Gemfile if you are okay with that (I'll add it to PR).

#319

### CODE REVIEW

File: app/views/catalog/_show_top_fields.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<% document = @document %>
+<% doc_presenter = show_presenter(document) %>
+<dl class="row">
+  <% document_show_fields(document).each do |field_name, field| -%>
+    <% if render_top_field? document, field, field_name %>
+      <dt class="col-3 text-right blacklight-<%= field_name.parameterize %>-label"><%= render_document_show_field_label(document, field: field_name).html_safe %></dt>
+      <% if doc_presenter.configuration.show_fields.fetch(field_name)[:helper_method] %>
```
Comment (banukutlu) [inline app/views/catalog/_show_top_fields.html.erb]: @cdmo I think we need a check for multi-valued fields for fields like additional creators like here: https://github.com/psu-libraries/psulib_blacklight/blob/master/app/views/catalog/_show_default.html.erb#L7?

**Comment by banukutlu** [comment]
> WIP, meaning don't merge. But, I'd like a first review. What's missing is tests. Also missing is transforming the Bookmark action into a member of the record-level action link button bar.
> 
> Additional change: I moved the pivot facet sass out of psulib_icons.
> 
> Semi-related: I started using better_errors, it is _better_ than web_console. Thinking we should replace it in the Gemfile if you are okay with that (I'll add it to PR).
> 
> #319

@cdmo I think these changes looks good. A minor thing for multi valued fields for top section but also I know this is still WIP. But generally how you implemented the items view sections is looking good to me 👍

**Comment by cdmo** [comment]
Cool, thanks @banukutlu. I'll return to working on it with this general approach then.

File: app/views/catalog/_show_top_fields.html.erb (hunk 1 above)
Comment (cdmo) [reply app/views/catalog/_show_top_fields.html.erb]: I refactored this a bunch. Can you [take a look when you have a chance](https://github.com/psu-libraries/psulib_blacklight/pull/336/commits/0c362e5c7ae16ee2b0bad896c91c3f2072e604a6)?

Comment (banukutlu) [reply app/views/catalog/_show_top_fields.html.erb]: these changes looks great 👌 way cleaner 👍

**Pull request merged**